### PR TITLE
display list of previous tasks in the order it finished

### PR DESCRIPTION
**Pull request by avelynhc**
- `task_detail` page now can display the tasks in descending order
- change the finish_time of unfinished task to 0 to compare the value of finish_time between the tasks in `task_detail.py` in backend by using `sort` function

### CODE REVIEW

File: backend/resources/task_detail.py [hunk 1]
```diff
@@ -140,5 +140,9 @@ def get(self):
             return {"message": "An error occured to get the task detail"}, 500
         if task_detail:
             task_list = [task.json() for task in task_detail]
+            for task in task_list:
+                if not task["finish_time"]: 
+                    task["finish_time"] = 0
+            task_list.sort(key=lambda x: x.get("finish_time"))
```
Comment (lyndonoc) [inline backend/resources/task_detail.py]: can we sort by start_time instead and use the database's sort instead of sorting this manually?

**Review by lyndonoc — CHANGES_REQUESTED** [review]
one comment

**Comment by avelynhc** [comment]
- I changed the way tasks are displayed based on start_time using database's sort as per your request
- pr for logout and adding /new are accidently added, please review those as well
- Gracias!

**Review by lyndonoc — APPROVED** [review]
LGTM

**Pull request merged**